easybuild/easyconfigs/r/Raptor/Raptor-2.0.16-GCCcore-12.3.0.eb

```
@@ -0,0 +1,54 @@
+##
+# This is a contribution from SIB Swiss Institute of Bioinformatics
+# Homepage:     https://www.sib.swiss/research-infrastructure/competence-centers/vital-it
+#
+# Authors::     Sebastien Moretti <sebastien.moretti@sib.swiss>
+#
+# Based on Raptor2 RPM spec file:
+# https://git.rockylinux.org/staging/rpms/raptor2/-/blob/r8/SPECS/raptor2.spec
+# The RPM patches have been integrated in the version 2.0.16
+#
+# Update: Petr Král (INUITS)
+##
+easyblock = 'ConfigureMake'
+
+name = 'Raptor'
+version = '2.0.16'
+homepage = 'https://librdf.org/raptor/'
+description = """Set of parsers and serializers that generate Resource Description Framework
+(RDF) triples by parsing syntaxes or serialize the triples into a syntax."""
+# software_license = 'LicenseLGPLv2.1 + LicenseGPLv2 + LicenseApachev2'
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+source_urls = ['https://download.librdf.org/source']
+sources = ['raptor2-%(version)s.tar.gz']
+checksums = ['089db78d7ac982354bdbf39d973baf09581e6904ac4c92a98c5caadb3de44680']
+
+builddependencies = [
+    ('Autotools',  '20220317'),
+    ('binutils',   '2.40'),
```

Existing easyconfig for `Raptor` has a recent version of `make` as build dependency, we should keep it (it's likely that older versions of `make` won't work, and `make-4.4.1-GCCcore-12.3.0.eb` is available already anyway)

```suggestion
    ('binutils',   '2.40'),
    ('make',   '4.4.1'),